src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs

```diff
@@ -33,8 +35,10 @@ public static class OnnxCatalog
         public static OnnxScoringEstimator ApplyOnnxModel(this TransformsCatalog catalog,
             string modelFile,
             int? gpuDeviceId = null,
-            bool fallbackToCpu = false)
-        => new OnnxScoringEstimator(CatalogUtils.GetEnvironment(catalog), modelFile, gpuDeviceId, fallbackToCpu);
+            bool fallbackToCpu = false,
+            IDictionary<string, int[]> shapeDictionary = null)
+        => new OnnxScoringEstimator(CatalogUtils.GetEnvironment(catalog), modelFile, gpuDeviceId, fallbackToCpu,
```

Comment:
I think this should be a breaking change. I am afraid API compat is not active for this assembly. I will open an issue about it and fix it in the meantime.
Unfortunately the signature of the function changes if you add another parameter. The only way to do this change is to add a new overload with the new parameter.

Reply:
There is already a PR on reactivating API compat, could you approve it and rebase afterwards?

---
In reply to: [301718819](https://github.com/dotnet/machinelearning/pull/3963#discussion_r301718819) [](ancestors = 301718819)

Reply:
Sure. Thanks.

---
In reply to: [301768225](https://github.com/dotnet/machinelearning/pull/3963#discussion_r301768225) [](ancestors = 301768225,301718819)